webapp/masterclasses.py

```diff
@@ -1,6 +1,16 @@
+import os
 from datetime import datetime
 import flask
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+
 from webapp.spreadsheet import get_sheet, MissingCredential
+from webapp.models import PreviousSession, UpcomingSession
+
+db_engine = create_engine(os.getenv("DATABASE_URL"))
```

Comment:
Another question about `sqlalchemy`, does it have connection pooling enabled by default? If so, does it have reasonable default settings like how many connections in a pool and timeout for waiting on a connection?

Reply:
sqlalchemy has connection pooling, and with reasonable defaults:
https://docs.sqlalchemy.org/en/20/core/pooling.html#connection-pool-configuration